libs/features/personalization/stage-entitlements.js

```diff
@@ -0,0 +1,12 @@
+const STAGE_ENTITLEMENTS = {
+  '5a5fd14e-f4ca-49d2-9f87-835df5477e3c': 'photoshop-any',
```

Comment:
Shouldn't we have parity between stage and prod entitlements? stage has fewer than prod.

Reply:
This is what has been given by the PM, I'll check with him but we can followup with another PR for that

Reply:
No worries, was just thinking this might cause confusion when switching between environments.